rules/macos/persistence_login_logout_hooks_defaults.toml

```diff
@@ -33,7 +33,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where event.type == "start" and
+process where host.os.name == "macos" and event.type == "start" and
```

Comment:
This failure seems to be a bug in the parsing logic.

error:
```
FAILED tests/test_all_rules.py::TestEndpointQuery::test_os_and_platform_in_query - AssertionError: 'host.os.name' not found in [] : 5d0265bf-dea9-41a9-92ad-48a8dcd05080 - Persistence via Login or Logout Hook -> missing required field for endpoint rule
```

No fields were parsed: `[]`

```
rule.name
'Persistence via Login or Logout Hook'

rule.id
'5d0265bf-dea9-41a9-92ad-48a8dcd05080'

rule.contents.data.query
'process where host.os.name == "macos" and event.type == "start" and\n process.name == "defaults" and process.args == "write" and process.args in ("LoginHook", "LogoutHook") and\n not process.args :\n       (\n         "Support/JAMF/ManagementFrameworkScripts/logouthook.sh",\n         "Support/JAMF/ManagementFrameworkScripts/loginhook.sh",\n         "/Library/Application Support/JAMF/ManagementFrameworkScripts/logouthook.sh",\n         "/Library/Application Support/JAMF/ManagementFrameworkScripts/loginhook.sh"\n       )\n'

rule.contents.data.ast
PipedQuery(first=EventQuery(event_type='process', query=Boolean(value=False)), pipes=[])

str(rule.contents.data.ast)
'process where false'
```

The original logic is parsed simply into `process where false`

Reply:
looks like certain line breaks are legal

![image](https://user-images.githubusercontent.com/16747370/220983029-fa3024f6-ab6e-4160-af97-3a28a411e142.png)

![image](https://user-images.githubusercontent.com/16747370/220983094-f5b21913-8c4f-4edf-a530-b6adf340f77a.png)